lnrpc/rpc.proto

```diff
@@ -1002,6 +1015,121 @@ message ListUnspentResponse {
+        */
+	HTLC_OFFERED_REVOKE = 3;
+
+        /*
+        A witness that allows us to sweep an HTLC output sent to us in the case
+        that the remote party broadcasts a revoked commitment state.
+        */
+	HTLC_ACCEPTED_REVOKE = 4;
+
+        /*
+        A witness that allows us to sweep an HTLC output that we extended to a
+        party, but was never fulfilled.  This HTLC output isn't directly on the
+        commitment transaction, but is the result of a confirmed second-level
+        HTLC transaction. As a result, we can only spend this after a CSV delay.
+        */
+	HTLC_OFFERED_TIMEOUT_SECOND_LEVEL = 5;
+
+        /*
+        A witness that allows us to sweep an HTLC output that was offered to us,
+        and for which we have a payment preimage. This HTLC output isn't
+        directly on our commitment transaction, but is the result of confirmed
+        second-level HTLC transaction. As a result, we can only spend this after
+        a CSV delay.
+        */
+        HTLC_ACCEPTED_SUCCESS_SECOND_LEVEL = 6;
+
+        /*
+        A witness that allows us to sweep an HTLC that we offered to the remote
+        party which lies in the commitment transaction of the remote party. We
+        can spend this output after the absolute CLTV timeout of the HTLC as
+        passed.
+        */
+	HTLC_OFFERED_REMOTE_TIMEOUT = 7;
+
+        /*
+        A witness that allows us to sweep an HTLC that was offered to us by the
+        remote party. We use this witness in the case that the remote party goes
+        to chain, and we know the pre-image to the HTLC. We can sweep this
+        without any additional timeout.
+        */
+	HTLC_ACCEPTED_REMOTE_SUCCESS = 8;
+
+        /*
+        A witness that allows us to sweep an HTLC from the remote party's
+        commitment transaction in the case that the broadcast a revoked
+        commitment, but then also immediately attempt to go to the second level
+        to claim the HTLC.
+        */
+	HTLC_SECOND_LEVEL_REVOKE = 9;
+
+        /*
+        A witness type that allows us to spend a regular p2wkh output that's
+        sent to an output which is under complete control of the backing wallet.
+        */
+	WITNESS_KEY_HASH = 10;
+
+        /*
+        A witness type that allows us to sweep an output that sends to a nested
+        P2SH script that pays to a key solely under our control.
+        */
+	NESTED_WITNESS_KEY_HASH = 11;
+}
+
+message PendingSweep {
+    // The outpoint of the output we're attempting to sweep.
+    OutPoint outpoint = 1 [json_name = "outpoint"];
+
+    // The witness type of the output we're attempting to sweep.
+    WitnessType witness_type = 2 [json_name = "witness_type"];
+
+    // The value of the output we're attempting to sweep.
+    uint32 amount_sat = 3 [json_name = "amount_sat"];
+
+    // The fee rate we'll use to sweep the output.
+    uint32 sat_per_byte = 4 [json_name = "sat_per_byte"];
```

Comment:
This is the current fee rate of the input that was either a mapped conf target or a manual fee rate. `sat_per_byte` already describes this is a fee rate so including it would seem redundant?

Reply:
It isn't important, but you do have `amount_sat`...

Reply:
My understanding is that there's a general pattern for amount fields throughout the RPC where we label them with `sat`/`msat` accordingly.